index.js

```diff
@@ -52,6 +47,37 @@ module.exports = function standardVersion (argv, done) {
   })
 }
 
+/**
+ * attempt to update the version # in a collection of common config
+ * files, e.g., package.json, bower.json.
+ *
+ * @param argv config object
+ * @param newVersion version # to update to.
+ * @return {string}
+ */
+var configsToUpdate = {}
+function updateConfigs (args, newVersion) {
+  configsToUpdate[path.resolve(process.cwd(), './package.json')] = false
+  configsToUpdate[path.resolve(process.cwd(), './bower.json')] = false
+  Object.keys(configsToUpdate).forEach(function (configPath) {
+    try {
+      var stat = fs.lstatSync(configPath)
+      if (stat.isFile()) {
+        var config = require(configPath)
+        var filename = path.basename(configPath)
+        config.version = newVersion
+        fs.writeFileSync(configPath, JSON.stringify(config, null, 2) + '\n', 'utf-8')
+        checkpoint(args, 'bumping version in ' + filename + ' from %s to %s', [config.version, newVersion])
+        // flag any config files that we modify the version # for
+        // as having been updated.
+        configsToUpdate[configPath] = true
+      }
+    } catch (err) {
+      if (err.code !== 'ENOENT') console.warn(err.message)
```

Comment:
@bcoe This PR is so cool, thanks for getting back into the project so strongly, I've been AWOL due to increased pressure at our startup but am trying to catch up too. 

I'm slightly concerned about the fact that any write errors will be ignored, e.g. if you have a `bower.json` you would presumably want to have it updated too. If the write fails, you'll just get a error but the git commits etc. will go through!

I haven't yet settled on an opinion on how this should be handled, but I think it's part of a bigger problem related to rolling back changes if an error occurs during the whole run..